problems/removeNumbersAtOddIndices.js

```diff
@@ -1,9 +1,19 @@
 /** Remove all numbers at odd indices from an array
-* @param {number[]} arr - The input array
-* @returns {number[]} - An array removing all elements initially appearing at an odd index
-*/
-function removeNumbersAtOddIndices() {
+ * @param {number[]} arr - The input array
+ * @returns {number[]} - An array removing all elements initially appearing at an odd index
+ */
+//function removeNumbersAtOddIndices() {}
+//[1, 2, 3, 4, 5, 6, 7, 8, 9]; >>> [1, 3, 5, 7, 9,]
+// for in loop gives the index
 
-}
+const removeNumbersAtOddIndices = (arr) => {
+	let oddsArr = [];
+	for (const num in arr) {
```

Comment:
Try using .filter for cleaner code
Remember we can use the index in the callback. 
Interesting use of for in loop. Call num i or idx to make it more clear what things are.